core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -657,6 +662,30 @@ class SparkSubmitSuite
     conf3.get(PYSPARK_PYTHON.key) should be ("python3.5")
   }
 
+  test("ambiguous archive mapping results in error message") {
+    val dir = Utils.createTempDir()
+    val archive1 = Paths.get(dir.toPath.toString, "first.zip")
+    val archive2 = Paths.get(dir.toPath.toString, "second.zip")
+    Files.createFile(archive1)
+    Files.createFile(archive2)
+    val jars = "/jar1,/jar2"                 // --jars
+    val files = "local:/file1,file2"          // --files
+    // --archives
```

Comment:
Unnecessary comment. I know the other test has them, but I'd just remove these from this new code, since they don't add any useful information.